packages-content-model/roosterjs-content-model-dom/lib/formatHandlers/utils/parseValueWithUnit.ts

```diff
@@ -41,6 +43,23 @@ export function parseValueWithUnit(
             case 'in':
                 result = num * PixelPerInch;
                 break;
+            case 'rem':
+                if (currentSizePxOrElement && typeof currentSizePxOrElement != 'number') {
+                    const doc = currentSizePxOrElement.ownerDocument;
+                    const htmlRoot = doc.querySelector('html');
```

Comment:
can we put the result of root font size into editorContext, since during the calculation it should not be changed, then we don't need to calculate it multiple times.